FBTweak/_FBEditableTweakDictionaryViewController.h

```diff
@@ -9,24 +9,24 @@
 
 #import <UIKit/UIKit.h>
 
-@class FBTweak;
+@protocol FBEditableTweak;
 
 /**
   @abstract Displays list of keys in a dictionary tweak.
  */
-@interface _FBTweakDictionaryViewController : UIViewController
+@interface _FBEditableTweakDictionaryViewController : UIViewController
 
 /**
   @abstract Creates a tweak dictionary view controller.
   @discussion This is the designated initializer.
```

Comment:
1. remove.
2. add NS_DESIGNATED_INITIALIZER
3. Check whether the other initializers are still valid, if not disable them.
same for `._FBEditableTweakArrayViewController`

Reply:
Fixed across the VCs (dict, arr, color)